src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header_block {
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  padding: 10px 0;
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.menu_list {
+  display: flex;
+}
+
+.menu_list .menu_item {
+  list-style: none;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.menu_link {
+  height: 60px;
+  font-family: "Roboto", Arial, sans-serif;
+  color: #000;
+  font-weight: 500;
+  line-height: 60px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+}
+
+.menu_link:hover {
+  color: #00acdc;
+}
+
+.menu_list .menu_item:not(:first-child) .menu_link {
+  margin-left: 20px;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

```suggestion
.is-active,
.menu_link:hover {
  color: #00acdc;
}
```
Ці стилі дублюються. Можна об'єднати